index.bs

```diff
@@ -909,58 +908,75 @@ partial interface HTMLIFrameElement {
+    (<var>origin</var>), and a [=response=] (<var>response</var>), this algorithm returns a new
     <a>Feature Policy</a></p>
     <ol>
-      <li><a>Initialize <var>document</var>'s
-      Feature Policy</a></li>
-      <li>Let <var>inherited policy</var> be <var>document</var>'s Feature
-      Policy's <a>inherited policy</a>.</li>
-      <li>Let <var>declared policy</var> be a new ordered map.</li>
+      <li>Let <var>policy</var> be the result of running <a>Create a Feature Policy for a browsing
+      context</a> given <var>browsingContext</var>, and <var>origin</var>.</li>
       <li>Let <var>d</var> be the result of running <a
       href="#process-response-policy">Process response policy</a> on
-      <var>response</var> and <var>document</var>'s origin.</li>
+      <var>response</var> and <var>origin</var>.</li>
       <li>For each <var>feature</var> → <var>allowlist</var> of <var>d</var>:
         <ol>
-          <li>If <var>inherited policy</var>[<var>feature</var>] is true, then
-          set <var>declared policy</var>[<var>feature</var>] to
+          <li>If <var>policy</var>'s <a>inherited policy</a>[<var>feature</var>] is true, then
+          set <var>policy</var>'s <a>declared policy</a>[<var>feature</var>] to
           <var>allowlist</var>.</li>
         </ol>
       </li>
-      <li>Let <var>policy</var> be a new <a>feature policy</a>, with inherited
-      policy <var>inherited policy</var> and declared policy <var>declared
-      policy</var>.
-      </li>
+      <li>Return <var>policy</var>.</li>
+    </ol>
+  </section>
+  <section>
+    <h3 id="algo-initialize-from-response"><dfn export
+    id="initialize-from-response">Initialize <var>document</var>'s Feature
+    Policy from <var>response</var></dfn></h3>
+    <p>Given a [=response=] (<var>response</var>) and a <a>Document</a>
+    (<var>document</var>), this algorithm populates <var>document</var>'s
+    <a>Feature Policy</a></p>
+    <ol>
+      <li>Let <var>policy</var> be the result of running <a>Create a Feature Policy for a browsing
+      context from response</a> given <var>document</var>'s <a>browsing context</a>, and <var>document</var>'s
+      <a>origin</a>, and <var>response</var>.</li>
       <li>
         Set <var>document</var>’s <a href="https://html.spec.whatwg.org/multipage/dom.html#concept-document-feature-policy">feature policy</a> to <var>policy</var>.
       </li>
     </ol>
   </section>
   <section>
-    <h3 id="algo-define-inherited-policy"><dfn>Define an inherited policy for
-    <var>feature</var> in <var>document</var></dfn></h3>
-    <p>Given a feature (<var>feature</var>) and a <a>Document</a>
-    (<var>document</var>), this algorithm returns the <a>inherited policy</a>
-    for that feature.</p>
+    <h3 id="algo-define-inherited-policy"><dfn id="define-inherited-policy">Define an inherited policy for
+    <var>feature</var>, and <a>origin</a> in <a>browsing context</a></dfn></h3>
```

Comment:
This should just be `<dfn...>Define an inherited policy for <var>feature</var> in <a>browsing context</a></dfn>` to match the name used in create-for-browsingcontext.